Health/src/main/java/com/sciencesquad/health/health/database/AbstractDataContext.java

```diff
@@ -0,0 +1,21 @@
+package com.sciencesquad.health.health.database;
+
+
+import android.content.Context;
+
+/**
+ * Created by danielmiller on 2/11/16.
+ *
+ * An abstract class for databases. They must do the following
+ *  - Initialize the database
+ *  - Update the database
+ *  - Query the database
+ */
+public abstract class AbstractDataContext {
```

Comment:
Does this really solve all the problems for any storage medium? How do I set or get values by key? What other DB operations can I perform?